src/common/Tile/index.js

```diff
@@ -177,16 +177,16 @@ export const DetailsTile = (
 				{
 					!description &&
 					<TileData>
-					<TileDataContent>
-						{
-							movieTile
-								? `No description available.`
-								: `No biography available.`
-						}
-					</TileDataContent>
+						<TileDataContent>
+							{
+								movieTile
+									? `No description available.`
+									: `No biography available.`
+							}
+						</TileDataContent>
 					</TileData>
 				}
-				{movieTile &&
+				{movieTile && movieTile.length > 0 && (
```

Comment:
movieTile is a boolean it is used only to render genre Tags on MovieDetails pages and not render them on PeopleDetails so we do not need that condition 🙂

Reply:
Thanks for the feedback! I've already made the necessary adjustments in the code. I removed the unnecessary condition for movieTile as you suggested. Your input is always appreciated! 🚀